src/signing/index.ts

```diff
@@ -142,7 +143,7 @@ export default class SignatureRequestManager {
       }
       return agg
     }, [])
-    // const { sigRequestHash, auxilary_data } = await
+
```

Comment:
that coment was sudo documentation to remind me whaats being returned by an adapter it's fine for now that you removed it maybe instead put a coment above it and say this is whats being returned from an adappter that we care about
```suggestion
// this is the named keys we care about from post sign. { sigRequestHash, auxilary_data }
```